roles/sushy_emulator/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_sushy_emulator"
+
+cifmw_sushy_emulator_driver: libvirt
+cifmw_sushy_emulator_ssh_algorithm: rsa
+cifmw_sushy_emulator_ssh_key_file: bmaas-ssh-key-id_rsa
+cifmw_sushy_emulator_ssh_key_size: 4096
```

Comment:
We have a top-level parameter, defaulting to `521`: `cifmw_ssh_keysize`.

You want to inherit from it:
`cifmw_sushy_emulator_ssh_key_size: "{{ cifmw_ssh_keysize | default(521) }}"` (and maybe rename to *_ssh_keysize?)